scripts/build/Jenkins/Jenkinsfile

```diff
@@ -926,6 +936,10 @@ try {
         if (!someBuildHappened) {
             currentBuild.result = 'NOT_BUILT'
         }
+        if("${currentBuild.currentResult}" == "SUCCESS" && IsPeriodicPipeline(pipelineName) && IsPullRequest(branchName)) {
+            currentBuild.result = 'UNSTABLE'
+            currentBuild.result.color = 'BLUE'
```

Comment:
May want to verify that this attribute is valid. It failed AR with this message, but the build was marked as successful:

```
Exception: org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: No such field found: field java.lang.String color
```

Reply:
Also I'm not sure it's possible to override the status color, I think Jenkins sets this based on the result. So only the `currentBuild.result = 'UNSTABLE'` should be required. But I'll need to check.

Reply:
I'm still testing this change, it seems that we cannot just set currentBuild.result.color. I needed to create a PR to test the PR build. Will update this soon.

Reply:
Updated this to use build description to indicate the actual build succeeded but regular PR build is required to merge the PR.